### PR TITLE
Update multi-vehicle-simulation.md

### PR DESCRIPTION
**Pull request by tensarflow**
The mult-UAV simulation with Gazebo needed a little more description. xacro was covered pretty good, where using sdf files where not documented at all.

### CODE REVIEW

File: en/simulation/multi-vehicle-simulation.md [hunk 1]
```diff
@@ -120,6 +120,8 @@ To add a third iris to this simulation there are two main components to consider
     > **Note** Be aware of which port is `src` and `dst` for the different endpoints.
 
 
+If you like to use the `SDF` models in the repository, you need to specify the UDP and TCP communication ports manually for each `SDF` file. So if you want to add multiple UAV's of the same model, you need to create a seperate `SDF` file for every UAV with its custom UDP and TCP ports. To add the TCP port in the `SDF` file, add `<mavlink_tcp_port>4560</mavlink_tcp_port>` under `<mavlink_udp_port>14561</mavlink_udp_port>`. The [mavros_posix_sitl.launch](https://github.com/PX4/Firmware/blob/master/launch/mavros_posix_sitl.launch) is a good starting point to use `SDF` files in the launch.
```
Comment (hamishwillee) [inline en/simulation/multi-vehicle-simulation.md:123]: @lamping7 Can you please confirm that this is correct? It sounds like what is implied by the rest of the doc, so my first thought was that it probably is :-)

**Comment by hamishwillee** [comment]
Thanks @tensarflow . I've asked Anthony to comment on this as he as worked on the area extensively.

**Comment by lamping7** [comment]
I'll take a look soon at the details. I just want to quickly point out https://github.com/PX4/Firmware/pull/12306 which removed the need to create copies of the same model.

**Comment by hamishwillee** [comment]
Thanks very much @lamping7 . Are you OK to look at fixes related to https://github.com/PX4/Firmware/pull/12306 after reviewing this PR? I can follow this up, but it probably makes more sense for you to do so.

**Comment by tensarflow** [comment]
I think that solves the problem. Thank you very much @lamping7 !!

**Comment by tensarflow** [comment]
Also I'd like to point out that I can't actually find any documentation about how to customize these roslaunch files. For example there are various `SDF` files of different vehicles but in order to use them you need to change in [launch/mavros_posix_sitl.launch](https://github.com/PX4/Firmware/blob/master/launch/mavros_posix_sitl.launch) line 14 and specify whatever vehicle you want to use. I can't see any documentation where that is mentioned.
I mean it may seem obvious but for a starter that can cost hours to find out (like it was in my case :D)

**Comment by lamping7** [comment]
You need to Google roslaunch. That's ROS basics. You don't modify the file to change the vehicle. You specify it as an argument when you run the roslaunch command.

**Comment by tensarflow** [comment]
Okay I see..

**Comment by lamping7** [comment]
Close in favor of https://github.com/PX4/Devguide/pull/825 @hamishwillee?

**Comment by hamishwillee** [comment]
Yes :-). Thanks @lamping7 - an @tensarflow see #825

**Pull request closed without merging**